Makefile

```diff
@@ -181,7 +181,7 @@ test-unit:
 .PHONY: test-e2e
 test-e2e: KUBECONFIG?=$(HOME)/.kube/config
 test-e2e:
-	go test -v -timeout=30m ./test/e2e/ --kubeconfig $(KUBECONFIG)
+	go test -v -timeout=1h ./test/e2e/ --kubeconfig $(KUBECONFIG)
```

Comment:
This is a large increase but we are having issues here even prior to this PR regularly. For example https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-monitoring-operator/1316/pull-ci-openshift-cluster-monitoring-operator-master-e2e-agnostic-operator/1425448228010070016 was the latest.

This change will likely slow the tests down a bit (~5 minutes), so I think it is better to double our timeout than to have 2 failed tests runs before a pass for example.